lib/buffer.js

```diff
@@ -107,7 +111,7 @@ function Buffer(arg, encodingOrOffset, length) {
         'If encoding is specified then the first argument must be a string'
       );
     }
-    return Buffer.allocUnsafe(arg);
+    return allocate(arg);
```

Comment:
Nevermind, reverted both changes for indirections to simplify reasoning about this PR (they seem to be negligible).